### DISCUSSION TITLE
HTML Türkçeleştirme Çalışması

### DISCUSSION THREAD

**Discussion by volkantash** [turkce-yazilim-konati/yazilimca#26] [Türkçe]
**Aşağıdakileri Türkçeleştiriyoruz.**

| İNGİLİZCE      | TÜRKÇE        | ÖNERİLER      | AÇIKLAMALAR          |
|---------------|--------------|---------------|-----------------------|
| ```<html></html>``` | seçilmedi | [```<kybd></kybd>```](61#discussioncomment-1161164) <br/> [```<ümbd></ümbd>```](61#discussioncomment-1159688) | **TARTIŞMALAR** <br>[Karşılık Arayışı](61)<br> **BİLGİLER** <br> [Mozilla - HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) |
| ```<base>``` | seçilmedi | ```<özek>``` <br/> ```<taban>``` <br/> ```<başat>``` | ÖZEK = Özbekistanda: çekirdek, esas, kök, nüve, öz gibi anlamlara gelirken Türkiye ve Azerbaycanda merkez türünde anlama gelir. Sözlüklere bakınız. BAŞAT = (KG) Kaynak demek <br> **BİLGİLER** <br>[Mozilla - base](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) |
|[head](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head)|```<baş></baş>```| ```<baş></baş>``` | |
|[link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link)|```<örk>```| ```<örk>, <bağ>, <köprü>, <çılbır>``` | Örk sözcüğünü araştırınız. |
|```<meta>```|seçilmedi| ```<ilgin>, <ilgen>, <öte>``` | "Onunla ilgili değme neñ" anlamına en yakın gördüğümüz sözcük ilgin. ilmek, ilgi, ilgili, ilişmek, ilişki, ilişkin, ilişkili (**Yeni türettiğimiz sözcük**). (Arı Türkçe ile _"Bu verinin **ilginveri**leri neler?"_) (Kırma Türkçe ile "_Bu datanın metadataları neler?_") |
|```<style></style>```|```<kılık></kılık>```|```<kılık></kılık>, <alım></alım>, <çalım></çalım>``` | ALIM ile ÇALIM sözcüklerini öneren: [Ahmet BOZKURT](https://www.facebook.com/ahmet.bozkurt.5283166/) |
|```<title></title>```|```<alınlık></alınlık>```| | Ana tartışma başlığı: https://github.com/turkce-yazilim-konati/yazilimca/discussions/55 |
|```<body></body>```|seçilmedi| ```<ten></ten>,<eğin></eğin>,<vücut></vücut>``` | |
|```<address></address>```|```<yeradı></yeradı>```|```<yeradı></yeradı>``` | |
|```<article></article>```|seçilmedi|```<yazıca></yazıca>, <makale></makale>```| |
|```<aside></aside>```|seçilmedi|```<yanda></yanda>, <yanlık></yanlık>``` | |
|```<footer></footer>```|seçilmedi|```<ayakçı></ayakçı>```| |
|```<header></header>```|seçilmedi|```<başçı></başçı>,<başlık></başlık>``` | |
|```<h1></h1>~<h6></h6>```|seçilmedi|```<b1></b1>~<b6></b6>```| |
|```<hgroup></hgroup>```|seçilmedi|```<başlıkçatamağı></başlıkçatamağı>, <başlıktoparı></başlıktoparı>, <başlıkkümesi></başlıkkümesi>``` | |
|```<main></main>```|```<ana></ana>```|```<ana></ana>```| |
|```<nav></nav>```|seçilmedi|```<gezinim></gezinim>```| |
|```<section></section>```|seçilmedi|```<kesim></kesim>```| |
|```<blockquote></blockquote>```| | | |
|```<dd></dd>```|seçilmedi|```<açıklamaayrıntıları></açıklamaayrıntıları>```| Description Details |
|```<div></div>```|seçilmedi|```<bölüm></bölüm>, <bölme></bölme>```| |
|```<dl></dl>```| | | Description List |
|```<dt></dt>```| | | Description Term |
|```<figcaption></figcaption>```| | | |
|```<figure></figure>```| | | |
|```<hr>```|seçilmedi|```<yataykural></yataykural>``` | Horizontal Rule |
|```<li></li>```|seçilmedi|```<dizme></dizme>, <dizim></dizim>``` | List |
|```<ol></ol>```| | | Ordered List |
|```<p></p>```| | | Paragraph |
|```<pre></pre>```| | | Preformatted Text |
|```<ul></ul>```| | | Unordered List |
|```<a></a>```|```<çapa></çapa>```| ```<çapa></çapa>``` | Anchor |
|```<abbr></abbr>```|seçilmedi|```<kısaltım></kısaltım>```| |
|```<b></b>```|seçilmedi|```<kalın></kalın>``` | |
|```<bdi></bdi>```| | | |
|```<bdo></bdo>```| | | |
|```<br>```|seçilmedi|```<kır>```| |
|```<cite></cite>```|seçilmedi|```<angı></angı>```| |
|```<code></code>```| | | |
|```<data></data>```|```<veri></veri>```|```<veri></veri>```| |
|```<dfn></dfn>```| | | |
|```<em></em>```|seçilmedi|```<vurgu></vurgu>```| |
|```<i></i>```|seçilmedi|```<eğik></eğik>```| |
|```<kbd></kbd>```| | | |
|```<mark></mark>```| | | |
|```<q></q>```| | | |
|```<rb></rb>```| | | |
|```<rp></rp>```| | | |
|```<rt></rt>```| | | |
|```<rtc></rtc>```| | | |
|```<ruby></ruby>```| | | |
|```<s></s>```|seçilmedi|```<üstüçizili></üstüçizili>``` | |
|```<small></small>```|seçilmedi|```<küçük></küçük>```| |
|```<span></span>```|seçilmedi|```<karış></karış>, <gıdım></gıdım>```| |
|```<strong></strong>```|seçilmedi|```<güçlü></güçlü>```| |
|```<sub></sub>```|seçilmedi|```<altınbetime></altınbetime>```| |
|```<sup></sup>```|seçilmedi|```<üstünbetime></üstünbetime>```| |
|```<time></time>```|seçilmedi|```<zaman></zaman>, <çağ></çağ>, <oğur></oğur>, <güngen></güngen>```| |
|```<u></u>```| | | |
|```<var></var>```|seçilmedi|```<değişken></değişken>```| |
|```<wbr></wbr>```|seçilmedi|```<sözkır></sözkır>```| |
|```<area></area>```|```<alan></alan>```|```<alan></alan>```| |
|```<audio></audio>```| | | |
|```<img>```| | | |
|```<map></map>```|```<harita></harita>```|```<harita></harita>```| |
|```<track>```| | | |
|```<video></video>```| | | |
|```<embed></embed>```| | | |
|```<iframe></iframe>```| | | |
|```<object></object>```|seçilmedi|```<nesne></nesne>``` | |
|```<param>```| | | |
|```<picture></picture>```| | | |
|```<source>```|seçilmedi|```<kaynak></kaynak>```| |
|```<svg></svg>```| | | |
|```<math></math>```|```<???></???>```| | |
|```<canvas></canvas>```|```<???></???>```| | |
|```<noscript></noscript>```|seçilmedi|```<betimeyok></betimeyok>```| |
|```<script></script>```|seçilmedi|```<betime></betime>```| |
|```<del></del>```|seçilmedi|```<silik></silik>```| |
|```<ins></ins>```|seçilmedi|```<eklenik></eklenik>```| |
|```<caption></caption>```|| | |
|```<col></col>```|seçilmedi|```<sütun></sütun>```| |
|```<colgroup></colgroup>```|seçilmedi|```<sütunçatamağı></sütunçatamaği>, <sütunkümesi></sütunkümesi>```| |
|```<table></table>```| | | |
|```<tbody></tbody>```| | | |
|```<td></td>```| | | |
|```<th></th>```| | | |
|```<thead></thead>```| | | |
|```<tr></tr>```| | | |
|```<button></button>```|seçilmedi| ```<düğme></düğme>``` | |
|```<datalist></datalist>```|seçilmedi|```<veridizme></veridizme>, <veridizim></veridizim>``` | |
|```<fieldset></fieldset>```| | | |
|```<form></form>```| | |
|```<input>```| seçilmedi | ```<girdi></girdi>``` | |
|```<label></label>```|seçilmedi|```<kolan></kolan>```| |
|```<legend></legend>```| | | |
|```<meter></meter>```| | | |
|```<optgroup></optgroup>```| | | |
|```<option></option>```| | | |
|```<output>```| seçilmedi | ```<çıktı></çıktı>``` | |
|```<progress></progress>```|seçilmedi|```<ilerleyiş></ilerleyiş>```| |
|```<select></select>```|```<seçme></seçme>```| | |
|```<textarea></textarea>```|```<yazıalanı></yazıalanı>```| | |
|```<details></details>```|```<ayrıntılar></ayrıntılar>```| | |
|```<dialog></dialog>```|```<ayta></ayta>```| | |
|```<menu></menu>```|```<???></???>```| | |
|```<summary></summary>```|```<???></???>```| | |
|```<slot></slot>```|```<???></???>```| | |
|```<template></template>```|```<???></???>```| | |
|```<acronym></acronym>```|```<???></???>```| | |
|```<applet></applet>```|```<???></???>```| | |
|```<basefont>```|```<???></???>```| | |
|```<bgsound>```|```<ardırasesi></ardırasesi>```| | |
|```<big></big>```|```<büyük></büyük>```| | |
|```<blink></blink>```|```<kırpış></kırpış>```| | |
|```<center></center>```|```<orta></orta>```| | |
|```<command></command>```|```<komut></komut>```| | |
|```<content></content>```|```<içerik></içerik>```| | |
|```<dir></dir>```|```<???></???>```| | |
|```<element></element>```|```<???></???>```| | |
|```<font>```|```<???></???>```| | |
|```<frame>```|```<katman></katman>```| | |
|```<frameset></frameset>```|```<katmankurma></katmankurma>```| | |
|```<image></image>```|```<???></???>```| | |
|```<isindex>```|```<dizinmi></dizinmi>```| | |
|```<keygen>```|```<açarcan></açarcan>```| | |
|```<listing>```|```<diziş></diziş>```| | |
|```<marquee></marquee>```|```<???></???>```| | |
|```<menuitem></menuitem>```|```<???></???>```| | |
|```<multicol></multicol>```|```<çoklusütun></çoklusütun>```| | |
|```<nextid>```|```<???></???>```| | |
|```<nobr>```|```<???></???>```| | |
|```<noembed></noembed>```|```<???></???>```| | |
|```<noframes></noframes>```|```<???></???>```| | |
|```<plaintext></plaintext>```|```<???></???>```| | |
|```<shadow></shadow>```|```<gölge></gölge>```| | |
|```<spacer></spacer>```|```<???></???>```| | |
|```<strike></strike>```|```<???></???>```| | |
|```<tt></tt>```|```<ıraktür></ıraktür>```| | |
|```<xmp></xmp>```|```<???></???>```| | |
|<img width=300/>|<img width=300/>|||

| İNGİLİZCE | TÜRKÇE  | ÖNERİLER | AÇIKLAMALAR |
|---------------|----------------------------------------|---|---|
|```lang=""```|```dilce=""```|  | Küresel |
|```accept=""```|```????=""```| | form, input |
|```accept-charset=""```|```????=""```| | form |
|```accesskey=""```|```????=""```| | Küresel  |
|```action=""```|```????=""```| | form |
|```align=""```|```????=""```| | applet, caption, col, colgroup, hr, iframe, img, table, tbody, td, tfoot , th, thead, tr  |
|```allow=""```|```????=""```| | iframe |
|```alt=""```|```????=""```| | applet, area, img, input |
|```async=""```|```????=""```| | script |
|```autocapitalize=""```|```????=""```| | Küresel |
|```autocomplete=""```|```????=""```| | Küresel |
|```autofocus=""```|```????=""```| | Küresel |
|```autoplay=""```|```????=""```| | Küresel |
|```background=""```|```????=""```| | Küresel |
|```bgcolor=""```|```????=""```| | Küresel |
|```border=""```|```????=""```| | Küresel |
|```buffered=""```|```????=""```| | Küresel |
|```capture=""```|```????=""```| | Küresel |
|```challenge=""```|```????=""```| | Küresel |
|```charset=""```|```????=""```| | Küresel |
|```checked=""```|```????=""```| | Küresel |
|```cite=""```|```????=""```| | Küresel |
|```class=""```|```????=""```| | Küresel |
|```code=""```|```????=""```| | Küresel |
|```codebase=""```|```????=""```| | Küresel |
|```color=""```|```tüs="", renk=""```| | Küresel |
|```cols=""```|```????=""```| | Küresel |
|```colspan=""```|```????=""```| | Küresel |
|```content=""```|```????=""```| | Küresel |
|```contenteditable=""```|```????=""```| | Küresel |
|```contextmenu=""```|```????=""```| | Küresel |
|```controls=""```|```????=""```| | Küresel |
|```coords=""```|```????=""```| | Küresel |
|```crossorigin=""```|```????=""```| | Küresel |
|```csp=""```|```????=""```| | Küresel |
|```data=""```|```veri=""```| | Küresel |
|```data-*=""```|```veri-=""```| | Küresel |
|```datetime=""```|```????=""```| | Küresel |
|```decoding=""```|```????=""```| | Küresel |
|```default=""```|```????=""```| | Küresel |
|```defer=""```|```????=""```| | Küresel |
|```dir=""```|```????=""```| | Küresel |
|```dirname=""```|```????=""```| | Küresel |
|```disabled=""```|```????=""```| | Küresel |
|```download=""```|```????=""```| | Küresel |
|```draggable=""```|```????=""```| | Küresel |
|```enctype=""```|```????=""```| | Küresel |
|```enterkeyhint=""```|```????=""```| | Küresel |
|```for=""```|```????=""```| | Küresel |
|```form=""```|```????=""```| | Küresel |
|```formaction=""```|```????=""```| | Küresel |
|```formenctype=""```|```????=""```| | Küresel |
|```formmethod=""```|```????=""```| | Küresel |
|```formtarget=""```|```????=""```| | Küresel |
|```headers=""```|```????=""```| | Küresel |
|```height=""```|```????=""```| | Küresel |
|```hidden=""```|```????=""```| | Küresel |
|```high=""```|```yüksek=""```| | Küresel |
|```href=""```|```????=""```| | Küresel |
|```hreflang=""```|```????=""```| | Küresel |
|```http-equiv=""```|```????=""```| | Küresel |
|```icon=""```|```????=""```| | Küresel |
|```id=""```|```adlık=""```| | Küresel |
|```importance=""```|```????=""```| | Küresel |
|```integrity=""```|```????=""```| | Küresel |
|```intrinsicsize=""```|```????=""```| | Küresel |
|```inputmode=""```|```????=""```| | Küresel |
|```ismap=""```|```????=""```| | Küresel |
|```itemprop=""```|```????=""```| | Küresel |
|```keytype=""```|```????=""```| | Küresel |
|```kind=""```|```????=""```| | Küresel |
|```label=""```|```????=""```| | Küresel |
|```lang=""```|```dilce=""```| | Küresel |
|```language=""```|```dilce=""```| | Küresel |
|```loading=""```|```????=""```| | Küresel |
|```list=""```|```????=""```| | Küresel |
|```loop=""```|```????=""```| | Küresel |
|```low=""```|```????=""```| | Küresel |
|```manifest=""```|```????=""```| | Küresel |
|```max=""```|```????=""```| | Küresel |
|```maxlength=""```|```????=""```| | Küresel |
|```minlength=""```|```????=""```| | Küresel |
|```media=""```|```????=""```| | Küresel |
|```method=""```|```????=""```| | Küresel |
|```min=""```|```????=""```| | Küresel |
|```multiple=""```|```????=""```| | Küresel |
|```muted=""```|```????=""```| | Küresel |
|```name=""```|```????=""```| | Küresel |
|```novalidate=""```|```????=""```| | Küresel |
|```open=""```|```????=""```| | Küresel |
|```optimum=""```|```????=""```| | Küresel |
|```pattern=""```|```????=""```| | Küresel |
|```ping=""```|```????=""```| | Küresel |
|```placeholder=""```|```????=""```| | Küresel |
|```poster=""```|```????=""```| | Küresel |
|```preload=""```|```????=""```| | Küresel |
|```radiogroup=""```|```????=""```| | Küresel |
|```readonly=""```|```????=""```| | Küresel |
|```referrerpolicy=""```|```????=""```| | Küresel |
|```rel=""```|```????=""```| | Küresel |
|```required=""```|```????=""```| | Küresel |
|```reversed=""```|```????=""```| | Küresel |
|```rows=""```|```????=""```| | Küresel |
|```rowspan=""```|```????=""```| | Küresel |
|```sandbox=""```|```????=""```| | Küresel |
|```scope=""```|```????=""```| | Küresel |
|```scoped=""```|```????=""```| | Küresel |
|```selected=""```|```????=""```| | Küresel |
|```shape=""```|```????=""```| | Küresel |
|```size=""```|```????=""```| | Küresel |
|```sizes=""```|```????=""```| | Küresel |
|```slot=""```|```????=""```| | Küresel |
|```span=""```|```????=""```| | Küresel |
|```spellcheck=""```|```????=""```| | Küresel |
|```src=""```|```????=""```| | Küresel |
|```srcdoc=""```|```????=""```| | Küresel |
|```srclang=""```|```????=""```| | Küresel |
|```srcset=""```|```????=""```| | Küresel |
|```start=""```|```????=""```| | Küresel |
|```step=""```|```adım=""```| | Küresel |
|```style=""```|```kılık=""```| | Küresel |
|```summary=""```|```????=""```| | Küresel |
|```tabindex=""```|```????=""```| | Küresel |
|```target=""```|```????=""```| | Küresel |
|```title=""```|```alınlık=""```| | Küresel |
|```translate=""```|```çeviri=""```| | Küresel |
|```type=""```|```tüspür="", tür=""```| | Küresel |
|```usemap=""```|```????=""```| | Küresel |
|```value=""```|```değer=""```| | Küresel |
|```width=""```|```genişlik=""```| | Küresel |
|```wrap=""```|```????=""```| | Küresel |

**Comment by volkantash** [comment]
STYLE için bence en iyi karşılık KILIK
```
<sytle></style>
<kılık></kılık>
```

**Comment by volkantash** [comment]
```
<head></head> 
<baş></baş>

<button></button>
<düğme></düğme>
```

**Comment by volkantash** [comment]
meta = ed, davar (Süer Eker'den öneri)

**Comment by lemreaskin** [comment]
Merhabalar, bence Türkçeleştirme yaparken çok fazla kişinin bilmediği Türkçe'nin özündeki kelimeleri kullanmak yerine daha güncel olarak kullandığımız sözcükleri seçmek daha mantıklı olacaktır. Sonuçta öncelikli amaç insanları anlamını bildiği anahtar kelimeleri kullanarak programlamaya ısındırmak değil midir? Örneğin HTML Türkçeleştirmesinde "link" karışıklığı olarak "<örk>" değil "<bağlantı>" kullanılması bence daha uygun olacaktır. Yine aynı şekilde Python Türkçeleştirmesinde ise "break" ifadesinin karşılığı olarak "sı" yerine "kır" veya "kırmak" ifadesi daha uygun olacaktır diye düşünüyorum. Sonuçta sizin değinmek istediğiniz nokta bu dili kullanacak kişilerin anahtar kelimlerin anlamlarını bilerek bu programlamaya ısınmasını sağlamaktır. Kişi kelimenin anlamını bilmedikten sonra o kelimenin Türkçe veya İngilizce olmasının bir anlamı kalmayacaktır. Bu noktalara dikkat ederek Türkçeleştirme yapılması gerektiğini düşünüyorum. Teşekkürler.

**Reply by volkantash** [reply]
Böyle güzel bir eleştiri okuduğumda mutlu oluyorum. :)

Size bir konudan söz etsiyorum. Batıda bir dil yürütümü var. Her yerde yeni sözcükler kullanarak kişinin bilip bilmemesine bakmazlar, karamazlar. Öğrenmesi için zorlarlar. Onlarda işe yarıyor ise bizde de işe yarar. 

Link için kimse bağlantı yazmaz, demez. Link koptu gibi bir anlam, kavram yok. Bağlantı sözcüğünde süreklilik anlamı vardır. **İnternet bağlantısı koptu** gibi sözlerde karşımıza çıkar. Link sözcüğü bağlanmak için bir geçiş aracıdır. Ancak bu bağlantı değildir. Anlamca tutmuyor üzgünüz. Öte yandan bağlantı sözcüğünü kısaltıp bağ diyorlar artık. Örk sözcüğünü bilmiyorlar mı? Bu durumda sözcüğü öğretmek için belgelerimizde, içeriklerimizde örk sözcüğü ile ilgili bilgiler vereceğiz demektir. Bilen var mı? Var... Sayısızca.

Öte yandan bu çalışma yalnızca Türkiye'yi kapsamayacak. Kazak, Özbek, Kırgız, Türkmen, Oğuz, Gagauz ve daha nice kardaşlarımız da söz konusu.

Sımak eylemi İstanbuldan ötürü medya diline alınmamış ancak sımak, sınmak gibi eylemler Türkiye'de ve öbür Türkçelerde kullanılıyor. Örneğin makas anlamına gelen bir sözcük vardır. SINDI denir. Bu sözcüğü bir iki kez işittikten sonra aklınızda kalır. SI- ya da SIN- dediğimizde de bu sözcüğün ne olduğunu hatırlamak kolaylaşır.

Yaygın bilinen sözcükleri seçmeyi ben de isterim ancak kavramlar söz konusu, başka açılar söz konusu. Belki bugün dediğiniz gibi algılanır ama 3 yıla kalmaz sözcüklere alıştıklarında iş değişir. Şöyle düşünün. Batının bildiği ama bizim bilmediğimiz bir sözcüğü ağzamaya başladık. Eee biz bilmiyoruz. Batı diyor mu buna Türkler bu sözcüğü bilmiyor... Biz ne ediyoruz? Sözcüğü öğrenmek için yazılar okuyoruz. Kendi sözcüklerimiz söz konusu olunca İngilizce sözcüğe verdiğimiz emeği neden vermeyeceğiz?

Önerilerinizi vermekten çekinmeyin. Birlikte irdeleyerek bunun kararını vereceğiz. Yoksa milletin aklına YAZICI sözcüğü yatıyor diyerek PRINTER sözcüğünün karşılığı yazar ise WRITER sözcüğüne ne karşılık vereceğiz diye bakakalırız. Ki kaldık. print("Merhaba Dünya") 'nın Türkçesi bu anlayışa göre yaz("Merhaba Dünya") ya da ("Merhaba Dünya")yaz... Eee write("Merhaba Dünya") diye bir görev de var. Buna ne edeceğiz. Yazmak eylemi gitti şimdi... Hangi eylemi kullanacağız?

İşte ilk başta biz de sizin gibi düşünürdük ancak işin içine girdikçe kaz ayağını gördük. Bu durumlar karşısında izlememiz gereken bir öneriniz varsa dinlemek, okumak isterim. Bin sağlık göresiniz, teşekkürler.
